src/lib.rs

```diff
@@ -316,28 +320,29 @@ fn entry_version(input: &str) -> Res<&str, EntryItem> {
             opt(tag(":")),
             space1,
             opt(tag("\"")),
-            is_version,
+            take_till(|c| c == '"' || c == '\n' || c == '\r'),
+            // is_version,
             opt(tag("\"")),
             line_ending,
         )),
     )(input)
     .map(|(i, (_, _, _, _, _, _, _, version, _, _))| (i, EntryItem::Version(version)))
 }
 
-#[allow(clippy::needless_pass_by_value)]
-fn is_version<T, E: nom::error::ParseError<T>>(input: T) -> IResult<T, T, E>
-where
-    T: nom::InputTakeAtPosition,
-    <T as nom::InputTakeAtPosition>::Item: AsChar,
-{
-    input.split_at_position1_complete(
-        |item| {
-            let c: char = item.as_char();
-            !(c == '.' || c == '-' || c.is_alphanum())
-        },
-        nom::error::ErrorKind::AlphaNumeric,
-    )
-}
+// #[allow(clippy::needless_pass_by_value)]
+// fn is_version<T, E: nom::error::ParseError<T>>(input: T) -> IResult<T, T, E>
+// where
+//     T: nom::InputTakeAtPosition,
+//     <T as nom::InputTakeAtPosition>::Item: AsChar,
+// {
+//     input.split_at_position1_complete(
+//         |item| {
+//             let c: char = item.as_char();
+//             !(c == '.' || c == '-' || c.is_alphanum())
```

Comment:
Just by adding the new non alphanumeric chars would do the trick: ` !(c == '.' || c == '-' || c == '@' || c == '/' || c == ':' || c.is_alphanum())`

Reply:
@milesj would that suggestion be ok? or are you foreseeing more needed chars? If that's the case, I'd go with your current implementation and completely delete the `is_version` commented function.